pkg/handlers/v1/storage/db.go

```diff
@@ -0,0 +1,309 @@
+package storage
```

Comment:
This needs to be moved out from under the handlers/v1 package, and under the pkg folder. https://developer.atlassian.com/platform/security/guides/google-golang/service-repo/

Reply:
OK but I thought you said earlier this should be in a package named for what it implements..?

Reply:
Yep that's still true. So instead of `pkg/handlers/v1/storage` it should be `pkg/storage`

Reply:
The general rule is to name the implementation package after the domain type that it implements. We also have a soft rule around naming the files containing an implementation after the concrete's name. For example, if given a domain type of ```type Storage interface{}``` then we'd most expect to see `/pkg/storage/pgsql.go` or `/pgk/storage/redis.go`.